daemons/based/based_remote.c

```diff
@@ -449,6 +462,7 @@ cib_remote_msg(gpointer data)
              */
             return 0;
         } else if (rc != pcmk_rc_ok) {
+            free(ncd);
```

Comment:
Won't `cib_remote_connection_destroy()` still be called and free it?

Reply:
Yes, the fix actually got squashed into the next patch but I'll deal with that.